content/docs/graphql/read-only-tokens.md

```diff
@@ -39,9 +43,94 @@ Wild card matching is supported in the branch names using '\*' to match anything
 
 Wild card matching is useful for matching branches that have not been created yet and can be used for editorial workflows.
 
-### Make a fetch request using the API key
+### Making requests with the tina client
+#### Setting up the Tina client
+
+This client can be used for data fetching with Tina. It can be used on the client and server to make graphQL requests. 
+
+To set up first add a new file called `.tina/client.{js,ts}`
+
+```ts
+const branch = "main";
+const apiURL =
+  process.env.NODE_ENV == "development"
+    ? "http://localhost:4001/graphql"
+    : `https://content.tinajs.io/content/<Your Client ID>/github/${branch}`;
+
+// Token generated on app.tina.io
+export const client = createClient({
+    // default values
+    url: apiURL,
+    token: "Your Read Only Token generated above",
+})
+```
+When using "http://localhost:4001/graphql" the content will be queried from file system (This only works during devolvement or in CI) and when using `https://content.tinajs.io/content/<Your Client ID>/github/${branch}` the content will be queried from Tina cloud. 
+
+In most cases the `apiURL` is the same one that is used for editing. So instead of passing the passing `apiURL` the client can be passed.
```

Comment:
I think we should continue documenting passing `apiURL`  until we make using the data layer the default. 

Right now, using and passing the client is awkward if the user does not have the data layer enabled. (The will hit Github's Api limits)

Reply:
> I think we should continue documenting passing apiURL until we make using the data layer the default.

> Right now, using and passing the client is awkward if the user does not have the data layer enabled. (The will hit Github's Api limits)

We could just override the URL in getStaticProps / paths though right? 

If we want the experimental client contained as experimental, I just think it's fine keeping it contained here. It just caused me to have to think a little bit harder since the read-only-tokens, and the client are 2 new concepts outlined in this doc.